scss/utilities/_colours.scss

```diff
@@ -47,7 +47,7 @@ $colours: (
 		("thunder",				$thunder, 				$iron-light,	$iron-light)
 		("ship-grey", 			$ship-grey, 			$iron-light,	$iron-light)
 		("abbey", 				$abbey, 				$iron-light,	$iron-light)
-		("nevada", 				$nevada, 				$iron-light,	$iron-light)
+		("nevada", 				$nevada, 				$white,	        $white)
```

Comment:
@rav-pradhan do you want to follow this method for the new colour pallette as well?

Reply:
Just so I'm clear, do you mean the method of setting anchor/font colours automatically when a bg colour is set? I think it would be worthwhile to, at the very least for the background colours we're currently using from the new palette, and where we know what their respective font colour is